sros2/sros2/api/__init__.py

```diff
@@ -116,6 +122,25 @@ def check_openssl_version(openssl_executable):
         raise RuntimeError('need openssl 1.0.2 minimum')
 
 
+def write_key(
+    key,
+    key_path,
+    encoding=serialization.Encoding.PEM,
```

Comment:
I suggest making the kwargs keyword-only, it leads to an API that is easier to use and easier to change in a backward-compatible manner:

```python3
def write_key(
    key,
    key_path,
    *,
    encoding=serialization.Encoding.PEM,
    format=serialization.PrivateFormat.PKCS8,
    encryption_algorithm=serialization.NoEncryption()
```